rfcs/interop_2023.md

```diff
@@ -0,0 +1,100 @@
+
+### Proposals
+
+During the proposals period (September 15 to October 15), proposals are submitted as issues in the [Interop repo](https://github.com/web-platform-tests/interop). A proposal should be as specific as reasonably possible, but multiple proposals may be grouped later in the selection process.
+
+There are two kinds of proposals: focus areas where progress is measured using test results, and investigation efforts where progress is measuring against a task list. For both kinds of proposals, finishing them should meaningfully advance the state of interoperability for the web platform.
+
+#### Focus areas
+
+These proposals are for existing web platform features with good a spec and automated test suite which can be used to improve interoperability. Progress is measured by test results on wpt.fyi.
+
+The following information should be provided in a proposal:
+
+- Is this area a problem for web developers? For example, survey data and browser bug counts can help.
+- Is this area a problem for users?
+- How widely used is this feature? For example, [use counters](https://www.chromestatus.com/metrics/feature/popularity), [HTTP Archive](https://httparchive.org/) and [State of CSS](https://2021.stateofcss.com/en-US/features/) can help.
+- Specification for this feature.
+- A list of tests on wpt.fyi which will be used to score progress towards interoperability on the feature.
+
+#### Investigation efforts
+
+If an area doesn't have a good spec and test suite it's not possible to use wpt.fyi test results to judge progress. Instead, some other work is needed to bring the area to state where test results can be used.
+
+The following information should be provided in a proposal:
+
+- Is this area a problem for web developers? For example, survey data and browser bug counts can help.
+- Is this area a problem for users?
+- How widely used is this feature? For example, [use counters](https://www.chromestatus.com/metrics/feature/popularity), [HTTP Archive](https://httparchive.org/) and [State of CSS](https://2021.stateofcss.com/en-US/features/) can help.
+- Specification for this feature, if any.
+- A list of tasks which should be completed as part of the effort. This could be producing a report for a working group, a list of spec bugs which should be resolved, or tests/infrastructure which need to be created. A completed investigation effort should unblock the feature for inclusion as a regular focus area in the future.
+
+### Proposal selection
+
+Proposal selection (November 1 to November 30) will be done using a positions process similar to [Interop 2022](https://github.com/web-platform-tests/interop-2022/issues/38) but with some changes. The process is:
+
+- Each participating organization takes one of 3 positions on each proposal: support, neutral, or oppose.
+- A proposal is accepted if there is consensus, meaning support from at least two participating organizations and no opposition.
```

Comment:
On the "are two direct supports required" question that has already been somewhat discussed, I'd suggest that we adopt a somewhat looser policy. In particular I think that "oppose" should be absolute, but after that we should have the ability to come to mutual agreement on which proposals to accept based on the total amount we want to include, and any evidence of priority between the proposals without requiring people to formally change their position. For example, if we had 20 proposals, 5 with 2+ "support", 10 with 1 support and all neutral and 5 with all neutral, we might be unhappy with just taking 5 proposals total. But there are 15 other proposals that didn't get opposition and it would seem quite reasonable to say "we'll take 5 of the 10 with one clear support / rest neutral that people are more exicited about given discussion".

Reply:
That makes sense to me. The second round allows for some of this, but as written anything that has no support in round 1 is off the table. Do you think we should make explicit that we can revive such proposals if we have newfound support? It seems not very likely to happen in practice, but reasonable to account for the possibility.